src/helpers.ts

```diff
@@ -9,7 +9,9 @@ export function assertNever(value: never): never {
   throw new Error(`Unexpected value should never occur: ${value}`)
 }
 
-export function pick<O extends unknown, K extends keyof O>(
+type AllKeys<T> = T extends unknown ? keyof T : never
```

Comment:
this supports getting all the keys from a union type, which occurs in the UpdateBlock request params